content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -208,54 +92,12 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
 
 {{% /capture %}}
 
-## Confirm Installation
-
-To confirm successful installation of both a hypervisor and Minikube, you can run the following command to start up a local Kubernetes cluster:
-
-{{< note >}}
+## Starting your first cluster
 
-For setting the `--driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--driver` values is available in [specifying the VM driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
-
-{{< /note >}}
+From a terminal with administrator access (but not logged in as root), run:
```

Comment:
Administrator access? That wasn't listed as a prerequisite earlier. Also, I don't think it's always needed.

Reply:
I'm on the fence on this one. What privileges are necessary is entirely dependent on the hypervisor and host configuration. For instance, Hyper-V by default does need to be run in a terminal as administrator. Docker needs access to a `docker` group. Hyperkit needs `sudo`. 

I removed the verbiage here, but have added the admin access to the list of pre-requisites.